modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/display/context/ContentsSectionDisplayContext.java

```diff
@@ -126,7 +127,10 @@ public List<FDSActionDropdownItem> getFDSActionDropdownItems() {
 				"modal-permissions"));
 	}
 
+	public String[] getObjectFolderExternalReferenceCodes() {
+		return new String[] {"L_CMS_CONTENT_STRUCTURES"};
```

Comment:
we will need to create a constants for these

Reply:
ok, we can do it in the next iteration yes. thanks for the comment